icc-x-api/icc-calendar-item-x-api.ts

```diff
@@ -133,6 +133,23 @@ export class IccCalendarItemXApi extends iccCalendarItemApi {
       })
   }
 
+  /**
+   * Remove the following delegation objects from the 
+   * CalendarItem instance:
+   * delegations, cryptedForeignKeys, encryptionKeys, secretForeignKeys
+   * @param calendarItem The Calendar Item object
+   */
+  resetCalendarDelegationObjects(calendarItem: models.CalendarItemDto): models.CalendarItemDto {
+    const {
+      delegations,
+      cryptedForeignKeys,
+      encryptionKeys,
```

Comment:
No need to reset.
If the `calendarItem` was encrypted, removing this key would make it impossible to `encrypt` and `decrypt` it.